src/main/java/Main.java

```diff
@@ -8,6 +10,70 @@ public static void main(String[] args) {
     }
 
     public void run(Scanner scanner) {
-        // napisz swój program tutaj. Do wczytywania danych użyj przekazanego w parametrze scannera
+        List<Integer> listFromUser = getListFromUser();
+        readListFromBackwards(listFromUser);
+        sumNumbers(listFromUser);
+        int lowestNumber = getLowestNumber(listFromUser);
+        System.out.println("Najmniejsza liczba w liście to: " + lowestNumber);
+        int highestNumber = getHighestNumber(listFromUser);
+        System.out.println("Najwieksza liczba w liście to: " + highestNumber);
+    }
+
+    private int getHighestNumber(List<Integer> listFromUser) {
+        int highestNumber = 0;
+        for (Integer integer : listFromUser) {
+            if (integer > highestNumber){
+                highestNumber = integer;
+            }
+        }
+        return highestNumber;
+    }
+
+    private int getLowestNumber(List<Integer> listFromUser) {
+        int lowestNumber = listFromUser.get(0);
+        for (Integer integer : listFromUser) {
+            if (integer < lowestNumber){
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu